UI/window-basic-main.cpp

```diff
@@ -1507,6 +1507,8 @@ bool OBSBasic::InitBasicConfigDefaults()
 				  "StreamAudioEncoder", "aac");
 	config_set_default_string(basicConfig, "SimpleOutput",
 				  "RecAudioEncoder", "aac");
+	config_set_default_uint(basicConfig, "SimpleOutput", "SimpleRecTracks",
```

Comment:
It's already in the "SimpleOutput", having "Simple" in the option name is quiet redundant.

Reply:
true

Reply:
done